runtime/calamari/src/lib.rs

```diff
@@ -755,7 +755,7 @@ impl cumulus_pallet_dmp_queue::Config for Runtime {
 
 parameter_types! {
 	// Rotate collator's spot each 6 hours.
-	pub const Period: u32 = 6 * HOURS;
+	pub const Period: u32 = 2 * MINUTES;
```

Comment:
It should be 6 hours, right?

Reply:
Yeah, this is forgotten from testing. Will fix.